File: algorithm/programmers/12903_middleLetter/solution.js
function solution(s) {
  var answer = '';

  if (s.length % 2 == 1) {
    answer = s[parseInt(s.length / 2)];
  } else {
    let ind = parseInt(s.length / 2);
    answer = s[ind - 1] + s[ind];
  }

  return answer;
}

File: algorithm/programmers/12903_middleLetter/Solution.java
public class Solution {

	public static String solution(String s) {
	      String answer = "";
	      
	      int len = s.length();
	      
	      if ( len%2 == 0) {
	    	  	answer += s.substring(len/2 -1, len/2+1);
	      } else {
	    	  	answer += s.substring(len/2);
	      }
	      
	      return answer;
	  }
	
	public static void main(String[] args) {
		
	}

}
